### PR TITLE
Add custom style for DrawerItem

### PR DESCRIPTION
**Pull request by naviocean**

### CODE REVIEW

**Comment by vonovak** [comment]
generally speaking, I'm against adding too many props to customize stuff, but this one imho makes sense (documentation needs to be added too).

**Comment by naviocean** [comment]
I've updated doc as you asked.

**Comment by vonovak** [comment]
@naviocean sorry if my comment confused you, but I don't have rights to merge PRs, I was just expressing my opinion. Maybe @kelset can take a look at this.

**Comment by kelset** [comment]
Hey everyone, I'll take a look at this properly this weekend, it looks ok overall but I am sleeping basically atm so I don't trust myself.

(btw @vonovak I think @spencercarli tried to reach out to you to up your role, maybe ping me or him on twitter for details)

**Comment by kelset** [comment]
So, I agree that this styling option needs to be provided but I think it's sort of necessary to clarify the "3 levels" of customization of the DrawerItems, because we would end up having:

* `style` = to style the overall "container" for the DrawerItems.
* `itemStyle` = to style the single item "overall" (ex, add a margin)
* `labelStyle` = to style in case of a text label (contained in the DrawerItem)

I think that then it's would be better to:

1. rename `style` to `itemsContainerStyle` (in both the file & docs) (which yes I realize it will be a "breaking change")
1. in the docs, move `itemStyle` above `labelStyle` and change the wording a bit to something like `style object for the single item, which can contain an Icon and/or a Label` - something along those lines

**Comment by kelset** [comment]
@naviocean I think you misunderstood my comment, it was `style` to be renamed, not `itemStyle`

**Comment by naviocean** [comment]
@kelset sorry for mistake, I've re-updated.

**Comment by kelset** [comment]
Just noting this for myself or @spencercarli when we make the next release notes that this will be a breaking API change (though certainly worth it) and we should notate that.

**Comment by kelset** [comment]
CicleCI failed because you haven't run prettier after the changes:

```js
/root/react-navigation/src/views/Drawer/DrawerNavigatorItems.js
  31:29  error  Insert `,`  prettier/prettier

✖ 1 problem (1 error, 0 warnings)
  1 error, 0 warnings potentially fixable with the `--fix` option.
```

File: src/views/Drawer/DrawerNavigatorItems.js [hunk 1]
```diff
@@ -26,8 +26,9 @@ type Props = {
   getLabel: (scene: DrawerScene) => ?(React.Element<*> | string),
   renderIcon: (scene: DrawerScene) => ?React.Element<*>,
   onItemPress: (info: DrawerItem) => void,
-  style?: ViewStyleProp,
-  labelStyle?: TextStyleProp,
+  itemsContainerStyle?: ViewStyleProp,
+  itemStyle?: ViewStyleProp,
+  labelStyle?: TextStyleProp
```
Comment (kelset) [inline src/views/Drawer/DrawerNavigatorItems.js]: I think you need to add a ',' at the end of `labelStyle?: TextStyleProp` to fix the prettier issue

**Comment by kelset** [comment]
Thanks a lot @naviocean :)

**Pull request merged**